web/controllers/expertController/index.ts

```diff
@@ -265,7 +265,7 @@ async function putExpert(wixId: string, student: Student, info: ApiPutExpert): P
 async function GetExpertiseTagEntities(tagNames: string[]): Promise<ExpertiseTag[]> {
     const entityManager = getManager();
 
-    const tags: ExpertiseTag[] = await entityManager.find(ExpertiseTag, { where: tagNames.map(t => ({ name: t }))});
+    const tags: ExpertiseTag[] = await entityManager.find(ExpertiseTag, { where: { name: In(tagNames)}});
```

Comment:
This suddenly wouldn't compile anymore.... Maybe it has to do with the updated Node version? 
The In() is equivalent to the tagNames.map() function in this case, right?

Reply:
Concrete error:
```
Error:(268, 11) TS2322: Type '{ name: string; }[]' is not assignable to type 'ExpertiseTag[]'.
  Type '{ name: string; }' is missing the following properties from type 'ExpertiseTag': id, expertData
 ```

Reply:
no repro. Probably something wrong with your build setup